com.sap.cloud.lm.sl.cf.persistence/src/main/java/com/sap/cloud/lm/sl/cf/persistence/query/providers/SqlFileQueryProvider.java

```diff
@@ -329,7 +330,7 @@ private String getSelectWithContentQuery() {
     }
 
     protected String getContentColumnName() {
-        return FileServiceColumnNames.CONTENT;
+        return Constants.CONTENT;
```

Comment:
Maybe have some prefix for File Entry fields in the Constant naming so they are easier to find/group?